langchain/src/chains/query_constructor/tests/query_chain.int.test.ts

```diff
@@ -33,6 +33,10 @@ test("Query Chain Test", async () => {
       new Comparison(Comparators.lt, "length", 90),
     ])
   );
+  const sq6 = new StructuredQuery(
+    "",
+    new Comparison(Comparators.eq, "isReleased", true)
+  );
```

Comment:
This isn't _technically_ a required test addition since I didn't modify the actual Comparison logic, but since I added support for `boolean` types in `Comparison`, I thought it was worthwhile to include it here just for posterity.